oracle/price-feeder/config/config.go

```diff
@@ -38,7 +38,7 @@ var (
 	ErrEmptyConfigPath = errors.New("empty configuration file path")
 
 	// SupportedProviders defines a lookup table of all the supported currency API
-	// providers.
```

Comment:
on lines 23-31, can there be a comment specifying that

Validators that provide data for Sei's native oracle pull price data from multiple providers. This is a sample list of providers that validators can pull price data from